test/metabase/search/fulltext/scoring_test.clj

```diff
@@ -76,6 +76,19 @@
       (is (get (search.scoring/scorers search-ctx) ranker-key) "The ranker is enabled")
       (map (juxt :model :id :name) (#'search.postgres/fulltext search-string search-ctx)))))
 
+(defn- search-no-weights
+  "Like search but with all weights set to 0."
+  [& args]
+  (binding [search.config/*default-weights* (update-vals @#'search.config/*default-weights* (constantly 0))]
+    (apply search* args)))
+
+(defn- search [& args]
+  (let [result (apply search* args)]
+    (is (not= (apply search-no-weights args)
```

Comment:
I like this so we don't accidentally setup the entries for success

Reply:
Nice idea, but what do you think of *only* setting the corresponding weight to zero?

Reply:
did that